versioning/BRM/extentmap.cpp

```diff
@@ -1557,86 +1412,23 @@ void ExtentMap::save(const string& filename)
             progress += err;
         }
     }
-    else   // this is the fstream version to be expired
-    {
-        ofstream out;
 
-        // Make em writes to disk use a buffer size of StrmBufSize bytes (instead of the default 8K)
-        const unsigned StrmBufSize = 1 * 1024 * 1024;
-        scoped_array<char> buf(new char[StrmBufSize]);
-        out.rdbuf()->pubsetbuf(buf.get(), StrmBufSize);
+    //allocdSize = fFLShminfo->allocdSize / sizeof(InlineLBIDRange);
```

Comment:
Just a quick cosmetic question? Do we really need to preserve the comments?

Reply:
Probably not.  :)